Lecture2/Task2/Task2L2.java

```diff
@@ -0,0 +1,35 @@
+import java.util.Scanner;
+
+public class Task2L2 {
+
+    private static final String MESSAGE_SELECTED = "You have selected a ";
+    private static final String MESSAGE_INVALID = "You have entered an invalid day's index";
+
+    public static void main ( String[] args ) {
+        Scanner in = new Scanner ( System.in );
+        System.out.print ( "Please enter an index of a day within a week (1-7): " );
+        byte day;
+        try {
+            day = in.nextByte ( );
+        }
+        catch (Exception e) {
+            System.out.println ( MESSAGE_INVALID );
+            return;
+        }
+        if (day == 1)
```

Comment:
think how you can achieve the task using array

Reply:
fixed